### ISSUE TITLE
Non-equivalence between unicode and glyph / significant whitespace

### ISSUE THREAD

**Issue by crmsnbleyd** [uiua-lang/uiua#21]
There seems to be a difference between the functions with and without whitespace. An example:

```
zipWithIndex ← ' ∶⇡⧻,
zipWithIndex (-) [1 2 5 79]

stack: [1 3 7 78]
```

here, the function works as expected, but if you remove the whitespace between the `'` and the rest of the function, it does not work

```
zipWithIndex ← '∶⇡⧻,
zipWithIndex (+) [1 2 5 79]

stack: 
[1 2 5 79]
[0 1 2 3]
+
```

However in both cases , substituting `bind` instead of `'`  gives the first answer. Curiously, it does not get substituted with `'` in either case.

I recognize this might be abuse of `bind` and thus out of scope. Tested on the online [pad](https://www.uiua.org/pad)

**Comment by kaikalii** [comment]
This is intended behavior that needs to be documented.

The problem is that there needs to be a nice way of using a modifier with functions from the stack rather than the functions that syntactically follow it.
The solution I came up with is that if a space immediately follows the modifier, it parses following terms as separate values.

I am open to alternative behaviors, but I chose this because it was short.

**Comment by crmsnbleyd** [comment]
@kaikalii , thanks for your work on the language! I agree that it should be documented.
However, that still leaves the issue of  `'∶⇡⧻,` and `bind∶⇡⧻,` having different semantics. Your inputs on that?

**Comment by kaikalii** [comment]
It is intended that functions with an ascii glyph spelling do not allow formatting from words. There's no reason to type `bind` when `'` is right there on the keyboard.

**Comment by goyalyashpal** [comment]
> that still leaves the issue of `'∶⇡⧻,` and `bind∶⇡⧻,` having different semantics. Your inputs on that?

just for the records, @crmsnbleyd this is fixed/added in #84
